source/jormungandr/jormungandr/street_network/street_network.py

```diff
@@ -63,9 +65,40 @@ def get_street_network_routing_matrix(
     def status(self):
         pass
 
+    def is_too_far(self, mode, request, pt_object_origin, pt_object_destination):
+        """
+        this function tests grossly if the distance between two coords is too far regarding
+        the give parameter (max_{mode}_direct_path_duration)
+
+        here is just a quick and stupid implementation, the actual street network connector may have more
+        information thus compute more specifically, if so, just feel free to overwrite this function
+        in the connector
+
+        :param mode:
+        :param request:
+        :param pt_object_origin:
+        :param pt_object_destination:
+        :return: bool
+        """
+        speed = utils.make_speed_switcher(request)[mode]
+        orig_coord = utils.get_pt_object_coord(pt_object_origin)
+        dest_coord = utils.get_pt_object_coord(pt_object_destination)
+        crowfly_distance = utils.crowfly_distance_between(orig_coord, dest_coord)
+
+        return (crowfly_distance / speed) > request['max_{}_direct_path_duration'.format(mode)]
```

Comment:
This won't work very well for `car` journeys computed by engine with "real" speed, we may lost a lot of journeys